web3.js/src/connection.ts

```diff
@@ -3603,7 +3603,7 @@ export class Connection {
     commitment?: Commitment,
   ): Promise<RpcResponseAndContext<SignatureResult>>;
 
-  /** @deprecated Instead, call `confirmTransaction` using a `TransactionConfirmationConfig` */
+  /** @deprecated Instead, call `confirmTransaction` and pass in `BlockheightBasedTransactionConfirmationStrategy` or `DurableNonceTransactionConfirmationStrategy`  */
```

Comment:
So, one thing that this introduces is the possibility that – when we add a third strategy – someone could forget to update this comment.

Does this work?

1. Extract `BlockheightBasedTransactionConfirmationStrategy | DurableNonceTransactionConfirmationStrategy` into a union type called `TransactionConfirmationStrategy`.
2. Refer to it here using `{@link TransactionConfirmationStrategy}`

I _think_ that should make the docs autolink to that type, where folks can see the up-to-date list of strategies.

Reply:
Hi @steveluscher thanks for the suggestion. I updated the PR. Hope this works. My comment describing `TransactionConfirmationStrategy` may need work. Thanks.